docs/metrics.md

```diff
@@ -0,0 +1,136 @@
+# Send Metrics
+The metrics collection and analysis plan for Send, a forthcoming Test Pilot experiment.
+
+## Analysis
+Data collected by Send will be used to answer the following high-level questions:
+
+- How do users
+	- How often? How many?
+	- What is the retention?
+	- What is the distribution of senders?
+- How do recipients interact with promotional UI elements?
+	- Are file recipients converted to file senders?
+	- Are non-Firefox users converted to Firefox users?
+- Where does it go wrong?
+	- How often are there errors in uploading or downloading files?
+	- What types of errors to users commonly see?
+	- At what point do errors affect retention?
+
+## Collection
+Data will be collected with Google Analytics and follow [Test Pilot standards](https://github.com/mozilla/testpilot/blob/master/docs/experiments/ga.md) for reporting.
+
+### Custom Metrics
+- `cm1` - the size of the file, in bytes.
+- `cm2` - the amount of time it took to complete the file transfer, in milliseconds. Only include if the file completed transferring (ref: `cd2`).
```

Comment:
Correct. GA limits the numbers of metrics and dimensions, so it befits you to reuse them in this way.